src/Dapr.Client/DaprClientGrpc.cs

```diff
@@ -2390,10 +2390,17 @@ public override async Task<DaprMetadata> GetMetadataAsync(CancellationToken canc
             try
             {
                 var response = await client.GetMetadataAsync(new Autogenerated.GetMetadataRequest(), options);
-                return new DaprMetadata(response.Id,
-                                        response.ActorRuntime.ActiveActors.Select(c => new DaprActorMetadata(c.Type, c.Count)).ToList(),
-                                        response.ExtendedMetadata.ToDictionary(c => c.Key, c => c.Value),
-                                        response.RegisteredComponents.Select(c => new DaprComponentsMetadata(c.Name, c.Type, c.Version, c.Capabilities.ToArray())).ToList());
+                if (response is null)
```

Comment:
Probably not necessary?  A successful gRPC method will *always* return a valid response object?  (It's properties may be `null`, but the object itself cannot be?)

Reply:
I _think_ you're right. This was more just an excess of caution in case I wasn't right about that. I can remove it if you think it unnecessary.